package/CHANGELOG

```diff
@@ -112,6 +112,8 @@ Changes
    numpy.testing.assert_allclose #4438)
 
 Deprecations
+ * The `guess_bonds`, `vdwradii`, `fudge_factor`, and `lower_bound` kwargs
+   are deprecated for Universe creation. (Issue #4756, PR #4757)
```

Comment:
Perhaps state what should be used instead? Such as "use Universe.to_guess(...) instead".